### PR TITLE
Updated to version 1.0.0

### PR DESCRIPTION
**Pull request by melund**

### CODE REVIEW

**Comment by melund** [comment]
I think Continuum have some server issue at the moment

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 package:
     name: prompt_toolkit
     version: "{{ version }}"
 
 source:
-    fn: prompt_toolkit-{{ version }}.tar.gz
-    url: https://pypi.python.org/packages/source/p/prompt_toolkit/prompt_toolkit-{{ version }}.tar.gz
-    md5: 516826aff6a09536037b6d02a9a3451d
+    git_url: https://github.com/jonathanslenders/python-prompt-toolkit
+    git_rev: {{ version }}
```
Comment (ocefpaf) [inline recipe/meta.yaml]: @melund can you use the tarball (`https://github.com/jonathanslenders/python-prompt-toolkit/archive/1.0.0.tar.gz`) and the sha256 (`35a7ae49d5f654ac3f8d1619572ff0ec37403c230399819cef962426b560a047`) here?

**Comment by scopatz** [comment]
This looks good to me other than the not-uploading and the url issue.

**Comment by ocefpaf** [comment]
I re-started the CIs and everything is passing now. Just waiting for a :+1: or :-1: on the source URL to merge.

**Comment by melund** [comment]
I have updated the URL. What is the reason not to use the git URL directly? Also, how do you get the checksum for the releases on Github without downloading?

**Comment by scopatz** [comment]
@melund I think the reason for not using the git url is that it pulls in the whole repo and thus increases download and build time. I have found that I needed to do a manual download and checksum to get the value.

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipe/meta.yaml]: Why not continue to use the PyPI URL?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
 package:
     name: prompt_toolkit
     version: "{{ version }}"
 
 source:
-    fn: prompt_toolkit-{{ version }}.tar.gz
-    url: https://pypi.python.org/packages/source/p/prompt_toolkit/prompt_toolkit-{{ version }}.tar.gz
-    md5: 516826aff6a09536037b6d02a9a3451d
+    fn: {{ version }}.tar.gz
+    url: https://github.com/jonathanslenders/python-prompt-toolkit/archive/1.0.0.tar.gz
```
Comment (scopatz) [inline recipe/meta.yaml]: This could be `https://github.com/jonathanslenders/python-prompt-toolkit/archive/{{ version }}.tar.gz` right?

**Comment by jakirkham** [comment]
Curious why the URL is being changed from PyPI at all.

**Comment by scopatz** [comment]
I am with @melund on this one @jakirkham. I don't like using the pypi urls because it is not from the actual source.

**Comment by scopatz** [comment]
or should I say, original source

**Comment by ocefpaf** [comment]
Test were already green an the latest changes should not change the results. Merging this. Thanks @melund!

**Comment by jakirkham** [comment]
> I am with @melund on this one @jakirkham. I don't like using the pypi urls because it is not from the actual source.

I'd like to discuss this more with all of you. However, it doesn't seem appropriate on this thread and honestly there are too many issues like this to keep track of. It would be better if we just add it to our agenda for the next general meeting.

**Comment by scopatz** [comment]
@jakirkham sounds good to me.

**Comment by jakirkham** [comment]
I added it under PyPI metadata and automation as I see having a standard here as being related to automating this effectively. Though it is a point I haven't talked about per se. If it looks like it is getting too out of hand as a subpoint, we can table it and make it a formal point.

**Pull request merged**